### ISSUE TITLE
Crash report 6.1: IAE in PeopleInviteFragment.isPrivateSite

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#4812]
```
Fatal Exception: java.lang.IllegalArgumentException: the bind value at index 2 is null
       at android.database.sqlite.SQLiteProgram.bindString(SQLiteProgram.java:176)
       at android.database.sqlite.SQLiteProgram.bindAllArgsAsStrings(SQLiteProgram.java:212)
       at android.database.sqlite.SQLiteDirectCursorDriver.query(SQLiteDirectCursorDriver.java:47)
       at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1454)
       at android.database.sqlite.SQLiteDatabase.queryWithFactory(SQLiteDatabase.java:1301)
       at android.database.sqlite.SQLiteDatabase.query(SQLiteDatabase.java:1172)
       at android.database.sqlite.SQLiteDatabase.query(SQLiteDatabase.java:1340)
       at org.wordpress.android.WordPressDB.getBlogForDotComBlogId(WordPressDB.java:934)
       at org.wordpress.android.ui.people.PeopleInviteFragment.isPrivateSite(PeopleInviteFragment.java:660)
       at org.wordpress.android.ui.people.PeopleInviteFragment.getDefaultRole(PeopleInviteFragment.java:279)
       at org.wordpress.android.ui.people.PeopleInviteFragment.onViewCreated(PeopleInviteFragment.java:123)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:998)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1163)
       at android.app.BackStackRecord.run(BackStackRecord.java:793)
       at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1552)
       at android.app.FragmentManagerImpl$1.run(FragmentManager.java:487)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:158)
       at android.app.ActivityThread.main(ActivityThread.java:7225)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
```

**Comment by nawabhussain** [comment]
@maxme can you share the scenario so that I can reproduce it ?

**Comment by maxme** [comment]
I don't have any scenario, but looking at the code the code, there is probably a null `dotComBlogId` passed to the  `PeopleInviteFragment.newInstance`

**Comment by daniloercoli** [comment]
Addressed in #4833
